src/main/kotlin/no/nav/syfo/aktivitetskrav/VurderStoppService.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.syfo.aktivitetskrav
+
+import no.nav.syfo.aktivitetskrav.domain.ExpiredVarsel
+import no.nav.syfo.database.DatabaseInterface
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.metric.COUNT_PERSONOPPGAVEHENDELSE_AKTIVITETSKRAV_EXPIRED_VARSEL_MOTTATT
+import no.nav.syfo.personoppgave.*
+import no.nav.syfo.personoppgave.domain.*
+import no.nav.syfo.personoppgavehendelse.domain.PersonoppgavehendelseType
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class VurderStoppService(
```

Comment:
Kanskje bedre å bruke `stans` i stedet for `stopp`, siden det er det vi kaller oppgaven?